assets/training/finetune_acft_hf_nlp/environments/acpt/context/finetune_run.py

```diff
@@ -429,15 +516,32 @@ def initiate_run():
 
     _initiate_run(completion_files_folder, model_selector_output,
                   preprocess_output, pytorch_model_folder, mlflow_model_folder)
-    cleanup(completion_files_folder, model_selector_output,
-            preprocess_output, pytorch_model_folder, mlflow_model_folder)
+
+    if is_main_process():
+        cleanup(completion_files_folder, model_selector_output,
+                preprocess_output, pytorch_model_folder, mlflow_model_folder)
+
+
+def parse_to_int(s):
+    """
+    To parse string to integer with default value in case of failure as one.

```

Comment:
why do you want to default to 1? Isn't it better to fail fast if node counts or gpus is not set or set to an invalid value.

Reply:
Why is this resolved?

Reply:
sorry , I missed to reply this . Reason I thought was lowest possible value is 1 , so keep that as default in case of parsing failure